www/templates/settings.html

```diff
@@ -7,6 +7,9 @@
          <ion-item ng-click="login()">
             Community Group Leader Login
          </ion-item>
+		 <ion-item ng-click="logout()">
```

Comment:
this will be active even when the user is not logged in. may be worth making it disappear when the user is not logged in and making the login button invisible when the user is  logged in...